libsolidity/parsing/Parser.cpp

```diff
@@ -811,6 +811,9 @@ ASTPointer<VariableDeclaration> Parser::parseVariableDeclaration(
 					case Token::CallData:
 						location = VariableDeclaration::Location::CallData;
 						break;
+					case Token::Code:
+						solUnimplemented("Code keyword as a data location specifier has not been implemented yet.");
```

Comment:
I was checking and for this point to be reached, `TokenTraits::isLocationSpecifier` needs to be modified to recognize `Token::Code`. Also, this throws an exception and I understood from #13368 that it should be a syntax error.